src/github-url-parser/.eslintrc.js

```diff
@@ -1,10 +1,26 @@
 module.exports = {
   extends: ['@senecacdot/eslint-config-telescope'],
-
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 2021,
+  },
   env: {
-    node: true,
     browser: true,
-    commonjs: true,
+    node: true,
     jest: true,
   },
+  plugins: ['@typescript-eslint'],
```

Comment:
The test uses `JS` right? Can we put this in `overrides`
```suggestion
    overrides: [
    {
      parser: '@typescript-eslint/parser',
      files: 'src/**/*.ts',
      plugins: ['@typescript-eslint'],
      rules: {
        '@typescript-eslint/no-unused-vars': 'off',
        '@typescript-eslint/explicit-function-return-type': 'off',
        '@typescript-eslint/explicit-module-boundary-types': 'off',
      },
    },
  ],
```